keda/templates/12-keda-deployment.yaml

```diff
@@ -35,11 +35,13 @@ spec:
         {{- if .Values.podIdentity.azureWorkload.enabled }}
         azure.workload.identity/use: "true"
         {{- end }}
-      {{- if .Values.podAnnotations.keda }}
       annotations:
```

Comment:
I would need to test this, but what if there are no annotations specified and it's basically just `annotations:`? I think it's fine but just want to verify